src/database/queries/proposalVersions/insertOne.sql

```diff
@@ -0,0 +1,22 @@
+INSERT INTO proposal_versions (
+  proposal_id,
+  application_form_id,
+  version
+) VALUES (
+  :proposalId,
+  :applicationFormId,
+  COALESCE(
+    (
+      SELECT MAX(pv.version) + 1
```

Comment:
There is probably a race condition here, but I am not sure what bad thing (if any) would happen even when it is encountered. It should just be a unique constraint violation on one of the transactions or something. And I doubt we will see such a thing any time soon.

Reply:
Oh good point -- does psql not prevent multiple writes at a time / is that why sequences are implemented the way they are 😬.

There is another instance of this type of logic (over in application forms) -- so I think it's already a problem that needs to be solved.  I'll create an issue to capture it.

Reply:
Captures in: https://github.com/PhilanthropyDataCommons/service/issues/109